styles/styles.css

```diff
@@ -267,12 +267,87 @@ blockquote {
   margin: var(--spacer-element-05) 0 var(--spacer-element-05);
 }
 
-main .columns-gap-50>div {
+main .column-content-top-border div>div {
+  margin-bottom: 32px;
+  max-width: 350px;
+}
+
+main .columns.columns-gap-50>div {
   gap: 50px
 }
 
-main .column-content-top-border div>div {
-  margin-bottom: 32px
+main .padding-bottom-40 {
```

Comment:
These are overly technical section styles an author will need to know and pick from to render out a specific design. Can authors pick and choose or are only certain combainations of padding allowed? Could terms such as `compact` or `spacious` be used to indicate how much padding should be applied to a section?